src/panels/config/backup/components/overview/ha-backup-overview-summary.ts

```diff
@@ -82,24 +94,50 @@ class HaBackupOverviewBackups extends LitElement {
       `;
     }
 
-    const lastSuccessfulBackup = this._lastSuccessfulBackup(this.backups);
+    const lastBackup = this._lastBackup(this.backups);
 
-    const lastAttempt = this.config.last_attempted_automatic_backup
+    const nextBackupDescription = this._nextBackupDescription(
+      this.config.schedule.state
+    );
+
+    // If no backups yet, show warning
+    if (!lastBackup) {
```

Comment:
It could be that the last attempt failed, we don't show that now.

Reply:
Right, we should move the last attempt test before that case.